chainer/__init__.py

```diff
@@ -29,6 +29,8 @@
 
 # import class and function
 # These functions from backends.cuda are kept for backward compatibility
+from chainer._runtime_info import get_runtime_info  # NOQA
```

Comment:
I think these lines should not be placed after the comment above.

Reply:
It should work because the comment explicitly states what the subjects it's talking about.
If the comment is to be moved to just in front of the subjects, I think these imports should be separated from other imports.

Reply:
OK.  I think it is better to move legacy things to https://github.com/niboshi/chainer/blob/d5cf9389835e2bf83857633b53daa5dd271459f5/chainer/__init__.py#L69 but it's not related to this PR.

Reply:
OK, I'll leave it as is for this PR.